.travis.yml

```diff
@@ -0,0 +1,47 @@
+cache: pip
+
+addons:
+  apt:
+    sources:
+      - pov-wkhtmltopdf
+    packages:
+      - expect-dev  # provides unbuffer utility
+      - poppler-utils # document index
+      - antiword # document index
+      - python-ldap # auth_ldap
+      - wkhtmltopdf # report_webkit
+
+language: python
+sudo: false
+
+python:
+  - "2.7"
+
+env:
+  - VERSION="10.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0"
+
+virtualenv:
+  system_site_packages: true
+
+before_install:
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+
+install:
+  - ln -s ${HOME}/build/${ODOO_REPO} ${HOME}/${ODOO_REPO#*/}-${VERSION}
+  - git clone --depth=1 https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
+  - pip install -Ur ${HOME}/maintainer-quality-tools/requirements.txt
+  - pip install -Ur ${HOME}/build/${ODOO_REPO}/requirements.txt
+  - pip install -q QUnitSuite flake8 coveralls pylint
+  - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
+  - export INCLUDE=$(getaddons.py -m ${HOME}/build/${ODOO_REPO}/openerp/addons ${HOME}/build/${ODOO_REPO}/addons)
+  - export EXCLUDE=hw_scanner,hw_escpos
```

Comment:
I think [you need to exclude also `theme_bootswatch`](https://travis-ci.org/OCA/OCB/builds/282180916#L1142), right @moylop260?